### DISCUSSION TITLE
Could you please produce a comparison with Nim, Zig, and possibly Terra and some others?

### DISCUSSION THREAD

**Discussion by mavavilj** [edubart/nelua-lang#184] [Ideas]
Could you please produce a comparison with Nim, Zig, and possibly Terra and some others?

I was thinking that I might jump to Nelua, when I was initially learning Nim. I perceived that the Nelua syntax might be even simpler than Nim's, while giving about the same features.

**Comment by Andre-LA** [comment]
About Terra, this comparison was made here:

[Comparison between Nelua and Terra ](https://github.com/edubart/nelua-lang/discussions/51)

**Comment by ghost** [comment]
Maybe we can also have comparison with Ravi  at some point, idk if this can be good idea...

**Comment by edubart** [comment]
In respect to all programming languages, I will be absent on any further comparisons with other languages. Apart from the fact that any comparison from me would be a little biased, I also I think most comparisons are unfair for all parts when regarding programming languages, as the scope of each language is broad therefore most comparisons are incomplete, furthermore most languages have different objectives and use cases. Also comparisons with other languages may attract people or discussions that are too opinionated that may develop into unfruitful conversations.

For people that want to compare new programming languages for your use case, the best way is to use all languages that aligns with your goals for a little while and decide later which one fits better.

**Comment by IcedQuinn** [comment]
I don't think I'm quite ready to abandon Nim. I have some years of code now in it and its the least aggravating compiled language I've used. That being said Nelua is definitely on my radar :eyes: because it is fascinating.
